workbench-core/datasets-ui/src/pages/new.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { useProjects } from '@aws/workbench-core-accounts-ui';
+import { BaseLayout } from '@aws/workbench-core-swb-common-ui';
+import { BreadcrumbGroupProps } from '@cloudscape-design/components';
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import React from 'react';
+import { createDataset } from '../api/datasets';
+import { NewDatasetForm } from '../components/CreateInternalDatasetForm';
+import { CreateDatasetForm } from '../models/Dataset';
+
+export const NewDatasetPage: NextPage = () => {
+  const router = useRouter();
+  const { projects, areProjectsLoading } = useProjects();
+
+  const breadcrumbs: BreadcrumbGroupProps.Item[] = [
+    {
+      text: 'Service Workbench',
```

Comment:
Is this meant to be a common component?

Reply:
Yes, as well as notifications. But that's out of the scope of the current PR